src/plugin.js

```diff
@@ -249,6 +249,7 @@ class Plugin {
             emulate: false,
             allowRootAndOutside: false,
             readWebpackConfiguration: false,
+            symlinks: true,
```

Comment:
Let's remove it. We will delete symlinks silently because it is default behavior of `rm -rf`.